nebula/ui/components/VPNComposerView.qml

```diff
@@ -0,0 +1,169 @@
+                    modelData instanceof VPNComposerBlockUnorderedList) return listBlock
+                if (modelData instanceof VPNComposerBlockButton)
+                    return composerBlock.style === VPNComposerBlockButton.Link ? linkButtonBlock : buttonBlock
+
+                console.error("Unable to create view for composer block of type: " + modelData)
+            }
+
+            function getTopMargin() {
+                if (modelData instanceof VPNComposerBlockTitle) {
+                    return VPNTheme.theme.vSpacingSmall
+                }
+
+                if (modelData instanceof VPNComposerBlockText) {
+                    switch(composer.view) {
+                    case VPNComposerView.View.Guide:
+                        return VPNTheme.theme.listSpacing * 0.5
+                    case VPNComposerView.View.Message:
+                    default:
+                        return VPNTheme.theme.vSpacingSmall
+                    }
+                }
+
+                if (modelData instanceof VPNComposerBlockOrderedList ||
+                        modelData instanceof VPNComposerBlockUnorderedList) {
+                    switch(composer.view) {
+                    case VPNComposerView.View.Guide:
+                        return VPNTheme.theme.listSpacing * 0.5
+                    case VPNComposerView.View.Message:
+                    default:
+                        return VPNTheme.theme.vSpacingSmall
+                    }
+                }
+
+                if (modelData instanceof VPNComposerBlockButton) {
+                    if(repeater.isFirstButton) {
+                        return VPNTheme.theme.vSpacingSmall * 2
+                    }
+                    return VPNTheme.theme.vSpacingSmall
+                }
+            }
+
+            Component {
+                id: titleBlock
+
+                VPNBoldInterLabel {
+                    text: loader.composerBlock.title
+                    font.pixelSize: VPNTheme.theme.fontSize
+                    lineHeight: VPNTheme.theme.labelLineHeight
+                    verticalAlignment: Text.AlignVCenter
+                    wrapMode: Text.Wrap
+                }
+            }
+
+            Component {
+                id: textBlock
+
+                VPNInterLabel {
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+
+                    text: loader.composerBlock.text
+                    font.pixelSize: VPNTheme.theme.fontSizeSmall
+                    color: VPNTheme.theme.fontColor
+                    horizontalAlignment: Text.AlignLeft
+                }
+            }
+
+            Component {
+                id: listBlock
+
+                VPNInterLabel {
+                    property string listType: loader.composerBlock && composerBlock.type === "olist" ? "ol" : "ul"
+                    property var tagsList: loader.composerBlock.subBlocks.map(subBlock => `<li>${subBlock}</li>`)
```

Comment:
For now it's not an issue but eventually we need to be sure that `subBlock` does not contain < or >. Or we need to escape a few chars.